pixl_core/src/core/uploader/_dicomweb.py

```diff
@@ -36,13 +36,15 @@ def __init__(self, project_slug: str, keyvault_alias: Optional[str]) -> None:
     def _set_config(self) -> None:
         # Use the Azure KV alias as prefix if it exists, otherwise use the project name
         az_prefix = self.keyvault_alias
-        az_prefix = az_prefix if az_prefix else self.project_slug
+        self.az_prefix = az_prefix if az_prefix else self.project_slug
```

Comment:
oh hello, fine for you to put this in as fixing it

Reply:
This has been reverted and is now being fixed in #381